blockchain/chain.go

```diff
@@ -1566,12 +1566,22 @@ func (b *BlockChain) maxBlockSize(prevNode *blockNode) (int64, error) {
 }
 
 // MaxBlockSize returns the maximum permitted block size for the block AFTER
-// the end of the current best chain.
+// the provided block hash.
 //
 // This function is safe for concurrent access.
-func (b *BlockChain) MaxBlockSize() (int64, error) {
+func (b *BlockChain) MaxBlockSize(hash *chainhash.Hash) (int64, error) {
+	// NOTE: The requirement for the node being fully validated here is strictly
+	// stronger than what is actually required.  In reality, all that is needed
+	// is for the block data for the node and all of its ancestors to be
+	// available, but there is not currently any tracking to be able to
```

Comment:
s / there is not currently any tracking / there is currently no tracking available

Reply:
This is consistent with the same text from multiple other locations, so I'd rather it not be inconsistent.

I'm not opposed to changing it to the suggestion, but I think that should be done in a separate PR so they can all be updated in tandem without polluting this PR with changes that don't specifically apply to it.